staging/src/k8s.io/kms/internal/plugins/base64/kms.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: base64-kmsv2-provider
```

Comment:
The name for this plugin might need to be changed to be generic. Maybe just call it mock-kmsv2-provider?